packages/e2e/lib/_extension_io.dart

```diff
@@ -0,0 +1,13 @@
+// Copyright 2019 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+/// The dart:io implementation of [registerWebServiceExtension].
```

Comment:
nit: enclose references to symbols in backticks, e.g. `dart:io`

Reply:
done.